src/main/java/network/oxalis/as4/outbound/MessagingProvider.java

```diff
@@ -23,7 +23,6 @@
 import java.util.Collection;
```

Comment:
There is No need to change anything in this class especially related to this Jakarta upgrade specific PR.